### PR TITLE
Dot names fix

### PR DESCRIPTION
**Pull request by Bugagazavr**

### CODE REVIEW

File: remote/gitlab/client/util.go [hunk 1]
```diff
@@ -5,8 +5,18 @@ import (
 	"strings"
 )
 
+var encodeMap = map[string]string{
+	".": "%252E",
```
Comment (tboerger) [inline remote/gitlab/client/util.go]: What about replacing slashes as before?

**Comment by bradrydzewski** [comment]
LGTM just needs a CLA

File: remote/gitlab/client/util.go (hunk 1 above)
Comment (Bugagazavr) [inline remote/gitlab/client/util.go]:
```go
value = url.QueryEscape(value)
```

Already do this

**Comment by Bugagazavr** [comment]
@bradrydzewski looks like cla acepted, this because my first commit has been sent with wrong credentionals, I use ammed to fix this, but badge cla is old, just look to checks

**Pull request merged**